yoti_test.go

```diff
@@ -1226,20 +1286,19 @@ func TestNewMultiValue(t *testing.T) {
 	assertIsExpectedDocumentImagesAttribute(t, documentImagesAttributeItems, multiValueAttribute.Anchors()[0])
 }
 
-func TestNestedMultiValue(t *testing.T) {
-	var innerMultiValueProtoValue []byte = createAttributeFromTestFile(t, "testattributemultivalue.txt").Value
+func TestInvalidMultiValueNotReturned(t *testing.T) {
+	var nilMultiValueItem = &yotiprotoattr.MultiValue_Value{
```

Comment:
Why is this _nilMultiValueItem_?

It looks like an invalid (incomplete) date.  And if the intention is to provide a date that cannot be parsed, you might as well just say 
```
Data:        []byte("invalid")
```

Reply:
Will change to be called invalidMultiValueItem, with the suggested value